CMakeLists.txt

```diff
@@ -112,19 +113,49 @@ if(UNIX AND NOT APPLE)
   target_link_libraries(kvspic rt)
 endif()
 
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/libkvspicClient.pc.cmake"
+  "${CMAKE_CURRENT_BINARY_DIR}/libkvspicClient.pc" @ONLY)
+
 add_library(kvspicClient STATIC ${PIC_CLIENT_SOURCE_FILES})
 target_link_libraries(kvspicClient)
 
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/libkvspicState.pc.cmake"
+  "${CMAKE_CURRENT_BINARY_DIR}/libkvspicState.pc" @ONLY)
+
 add_library(kvspicState STATIC ${PIC_STATE_SOURCE_FILES})
 target_link_libraries(kvspicState)
 
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/libkvspicUtils.pc.cmake"
+  "${CMAKE_CURRENT_BINARY_DIR}/libkvspicUtils.pc" @ONLY)
+
 add_library(kvspicUtils STATIC ${PIC_UTILS_SOURCE_FILES})
 target_link_libraries(kvspicUtils ${CMAKE_DL_LIBS} Threads::Threads)
 if(UNIX AND NOT APPLE)
   # rt needed for clock_gettime
   target_link_libraries(kvspicUtils rt)
 endif()
 
+install(
+  TARGETS kvspic kvspicClient kvspicState kvspicUtils
```

Comment:
Will this properly create a directory per project for the include files? I assume so as we would end up with name conflicts as all the public includes are named Include.h

Reply:
Yep, the prefix is configurable using `CMAKE_INSTALL_PREFIX`